mmv1/templates/terraform/examples/cloud_run_service_configuration.tf.erb

```diff
@@ -47,18 +55,34 @@ resource "google_cloud_run_service" "<%= ctx[:primary_resource_id] %>" {
             memory = "512Mi"
           }
         }
+        # [END cloudrun_service_configuration_limit_memory]
+        # [END cloudrun_service_configuration_limit_cpu]
+
+        # [START cloudrun_service_configuration_containers]
       }
+      # [END cloudrun_service_configuration_containers]
```

Comment:
this enclosure only contains a single bracket. Is that intentional?

Reply:
Yep - enclosures with the same tag-name will be concatenated together.